services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/core/docker_compose_utils.py

```diff
@@ -49,24 +49,28 @@ async def _write_file_and_spawn_process(
 
     This calls is intentionally verbose at INFO level
     """
-    async with write_to_tmp_file(yaml_content) as file_path:
-        cmd = command.format(file_path=file_path)
 
-        logger.debug("Runs %s ...\n%s", cmd, yaml_content)
+    logger.debug("Runs %s ...\n%s", command, yaml_content)
+
+    result = await async_command(
+        command=command, timeout=process_termination_timeout, pipe_as_input=yaml_content
+    )
+    debug_message = deepcopy(result._asdict())
```

Comment:
Why do you copy to log it? why you use the dict?  Moreover, `result` is a NamedTuple which is very "print friendly"

TIP: you can also define `__str__` in `CommandResult`